dotenv_linter/violations/values.py

```diff
@@ -62,3 +63,25 @@ class QuotedValueViolation(BaseFSTViolation):
 
     code = 301
     error_template = 'Found quoted value'
+
+
+@final
+class InvalidEOLViolation(BaseFSTViolation):
+    """
+    Restricts to use `/r/n` (CRLF) end-of-line.
+
+    Reasoning:
+        ???
```

Comment:
Mixing different end-of-line chars can lead to different hard-to-debug problems.